elements/pfe-accordion/src/pfe-accordion-header.scss

```diff
@@ -63,3 +64,38 @@ button[aria-expanded="true"] {
     @include pfe-accordion-variables($state: expanded, $theme: #{$context});
   }
 }
+
+/// ===========================================================================
+/// DISCLOSURE STYLES
+/// ===========================================================================
+
+:host([pfe-disclosure="true"]) {
+  button {
+    @include pfe-trigger($align: right);
+
+    &[aria-expanded="true"] {
+      --pfe-accordion--BorderLeftWidth: 4px;
+      padding-left: 47px !important;
```

Comment:
This definitely feels like a magic number, where are 47px and 21px coming from?  Consider how they will be impacted if someone changes the spacing variables that are impacting the padding on the header for example.  How can we use the variables like `container-spacer` to calculate a similar size?

Reply:
It's a bit of a magic number and it has to be exact or you'll see the accordion header text shifting about 4 px to the right when opened or on hover. I went with this instead:
`padding-left: calc(( #{pfe-var(container-spacer)} * 3 ) - 1px ) !important;`